examples/fuse_examples/imaging/classification/isic/isic_runner_ddp.py

```diff
@@ -0,0 +1,483 @@
+"""
```

Comment:
I suggest keeping only the isic example - but to modify the original isic example (the default can still be single GPU though)
It means that we will also remove the mnist example.
What do you think?

Reply:
Yeah sounds good.

While the mnist example is more readable, the ISIC one will be sufficient to follow for ones that wish to implement a DDP runner.